webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  Flex,
+  FlexItem,
+  GridItem,
+  Label,
+} from '@patternfly/react-core';
+
+import Skeleton from 'react-loading-skeleton';
```

Comment:
I meant the Patternfly skeleton :wink: 
```suggestion
import { Skeleton } from '@patternfly/react-core';
```